spec/core/v2/ics-004-packet-semantics/README.md

```diff
@@ -206,513 +196,111 @@ function channelCapabilityPath(portIdentifier: Identifier, channelIdentifier: Id
 
 ```typescript
-function packetReceiptPath(portIdentifier: Identifier, channelIdentifier: Identifier, sequence: uint64): Path {
-    return "receipts/ports/{portIdentifier}/channels/{channelIdentifier}/sequences/{sequence}"
+function packetReceiptPath(sourceId: bytes, sequence: uint64): Path {
+    return "receipts/channels/{sourceId}/sequences/{bigEndianUint64Sequence}"
 }
 ```
 
 Packet acknowledgement data are stored under the `packetAcknowledgementPath`:
 
 ```typescript
-function packetAcknowledgementPath(portIdentifier: Identifier, channelIdentifier: Identifier, sequence: uint64): Path {
-    return "acks/ports/{portIdentifier}/channels/{channelIdentifier}/sequences/{sequence}"
+function packetAcknowledgementPath(sourceId: bytes, sequence: uint64): Path {
+    return "acks/channels/{sourceId}/sequences/{sequence}"
 }
 ```
 
-### Versioning
-
-During the handshake process, two ends of a channel come to agreement on a version bytestring associated
-with that channel. The contents of this version bytestring are and will remain opaque to the IBC core protocol.
-Host state machines MAY utilise the version data to indicate supported IBC/APP protocols, agree on packet
-encoding formats, or negotiate other channel-related metadata related to custom logic on top of IBC.
-
-Host state machines MAY also safely ignore the version data or specify an empty string.
-
 ### Sub-protocols
 
 > Note: If the host state machine is utilising object capability authentication (see [ICS 005](../ics-005-port-allocation)), all functions utilising ports take an additional capability parameter.
 
-#### Identifier validation
-
-Channels are stored under a unique `(portIdentifier, channelIdentifier)` prefix.
-The validation function `validatePortIdentifier` MAY be provided.
-
-```typescript
-type validateChannelIdentifier = (portIdentifier: Identifier, channelIdentifier: Identifier) => boolean
-```
-
-If not provided, the default `validateChannelIdentifier` function will always return `true`.
+#### Counterparty Idenfitifcation and Registration 
 
-#### Channel lifecycle management
+A client MUST have the ability to idenfity its counterparty. With a client, we can prove any key/value path on the counterparty. However, without knowing which identifier the counterparty uses when it sends messages to us, we cannot differentiate between messages sent from the counterparty to our chain vs messages sent from the counterparty with other chains. Most implementations will not be able to store the ICS-24 paths directly as a key in the global namespace, but will instead write to a reserved, prefixed keyspace so as not to conflict with other application state writes. Thus the counteparty information we must have includes both its identifier for our chain as well as the key prefix under which it will write the provable ICS-24 paths.
```

Comment:
```suggestion
Each IBC channel end MUST have the ability to idenfity its counterparty. With a client, we can prove any key/value path on the counterparty. However, without knowing which identifier the counterparty uses when it sends messages to us, we cannot differentiate between messages sent from the counterparty to our chain vs messages sent from the counterparty with other chains. Most implementations will not be able to store the ICS-24 paths directly as a key in the global namespace, but will instead write to a reserved, prefixed keyspace so as not to conflict with other application state writes. Thus the counterparty information we must have includes both its identifier for our chain as well as the key prefix under which it will write the provable ICS-24 paths.
```